main/abuild/0001-abuild-properly-detect-package-size-on-btrfs.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/abuild.in b/abuild.in
+index 14955357af..38d2598b8f 100644
+--- a/abuild.in
++++ b/abuild.in
+@@ -812,7 +812,7 @@ prepare_metafiles() {
+ 	cd "$dir"
+ 	mkdir -p "$controldir"
+ 	local builddate=$(date -u "+%s")
+-	local size=$(du -sk | awk '{print $1 * 1024}')
++	local size=$(sync;du -sk | awk '{print $1 * 1024}')
```

Comment:
I'd rather not do this. This is a huge performance hit on our big build servers, which have several LXC containers running. Is there any way to fix btrfs? As minimum this should be conditional to if the filesystem is btrfs.